lists/api.py

```
@@ -1,52 +1,291 @@
     def get(self, request: Request, *args, **kwargs):
-        list_id = kwargs.get("list_id", None)
-        action = kwargs.get("action", None)
-        if list_id:
-            # Request pertaining to a specific list_id
+        lists = List.objects.all().select_related("owner").prefetch_related("admins", "upvotes").annotate(registrations_count=Count('registrations'))
+        account_id = request.query_params.get("account")
+        chain = request.query_params.get("chain", "NEAR")
+        lists = lists.filter(chain__name=chain)
+        if account_id:
+            try:
+                account = Chain.objects.get(name=account_id)
```

Ruff (0.14.1)

Undefined name `Chain`

(F821)